affinity/cpp-20/d0796r2.md

```diff
@@ -252,14 +292,20 @@ auto socketAllocator = eC.allocator(); // Retrieve an allocator to the closest m
 std::vector<int, decltype(socketAllocator)> v1(100, socketAllocator);
 std::generate(par.on(executor), std::begin(v1), std::end(v1), std::rand);
 ```
-*Listing 5: Example of allocating with affinity to an execution resource*
+*Listing 6: Example of allocating with affinity to an execution resource*
 
 The construction of an `execution_context` on a component implies affinity (where possible) to the given resource. This guarantees that all executors created from that `execution_context` can access the resources and the internal data structures requires to guarantee the placement of the processor.
 
 Only developers that care about resource placement need to care about obtaining executors and allocations from the correct `execution_context` object. Existing code for vectors and STL (including the Parallel STL interface) remains unaffected.
 
 If a particular policy or algorithm requires to access placement information, the resources associated with the passed executor can be retrieved via the link to the `execution_context`.
 
+### Binding to execution
+
+A *thread of execution* can be bound to a particular `execution_resource` for a particular *execution agent* by calling `this_thread::bind`. After which point the *execution resource* returned by `this_thread::get_resource` must be equal to the `execution_resource` provided to `this_thread::bind`. Subsequently a *thread of execution* can be unbound by calling `this_thread::unbind`.
```

Comment:
mention only if _can bind_

Reply:
@hcedwar Thanks I missed that, I've added a requirement that the `execution_resource` be capable of placing agents.
@Ruyk Thanks for pointing this out, I have added some wording which describes what happens when binding or unbinding is unssucessfull. For now, I avoided having it throw an exception just to avoid a barrier to implementation on architectures which don't support exceptions, though this may change in the future. Yes that's right, the idea is that after a thread of execution is successfully bound to an execution resource, any time you query that thread of execution for it's resource you should get the same resource you bound to it.